index.bs

```diff
@@ -228,18 +230,18 @@ The key words "MUST", "MUST NOT", "REQUIRED", "SHALL", "SHALL NOT", "SHOULD", "S
 :: See [=Authentication Assertion=].
 
 : <dfn>Attestation</dfn>
-:: Generally, a statement that serves to bear witness, confirm, or authenticate.
-    In the WebAuthn context, attestation is employed to attest to the provenance of an authenticator and the data it emits;
-    including, for example: credential IDs, credential key pairs, signature counters, etc. <dfn>Attestation information</dfn> is
-    conveyed in <dfn>attestation objects</dfn>.
-    See also [=attestation statement format=], and [=attestation type=].
+:: Generally, <em>attestation</em> is a statement serving to bear witness, confirm, or authenticate.
+    In the WebAuthn context, [=attestation=] is employed to <em>attest</em> to the <em>provenance</em> of an [=authenticator=]
```

Comment:
1. I'd like to be clearer about *who* is attesting to this sort of thing. Attestations aren't just handed down by God, they can be more or less trustworthy, and RPs ought to pay some attention to that.
2. Isn't it less the provenance of the data an authenticator emits (and stores, for private keys), than the security guarantees provided for that data?